html/js/manifest.js

```diff
@@ -1 +1,192 @@
-(()=>{"use strict";var e={},r={};function t(n){if(r[n])return r[n].exports;var o=r[n]={id:n,loaded:!1,exports:{}};return e[n].call(o.exports,o,o.exports,t),o.loaded=!0,o.exports}t.m=e,t.x=e=>{},t.n=e=>{var r=e&&e.__esModule?()=>e.default:()=>e;return t.d(r,{a:r}),r},t.d=(e,r)=>{for(var n in r)t.o(r,n)&&!t.o(e,n)&&Object.defineProperty(e,n,{enumerable:!0,get:r[n]})},t.g=function(){if("object"==typeof globalThis)return globalThis;try{return this||new Function("return this")()}catch(e){if("object"==typeof window)return window}}(),t.o=(e,r)=>Object.prototype.hasOwnProperty.call(e,r),t.r=e=>{"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},t.nmd=e=>(e.paths=[],e.children||(e.children=[]),e),(()=>{var e={929:0},r=[],n=e=>{},o=(o,l)=>{for(var a,i,[u,d,f,p]=l,s=0,h=[];s<u.length;s++)i=u[s],t.o(e,i)&&e[i]&&h.push(e[i][0]),e[i]=0;for(a in d)t.o(d,a)&&(t.m[a]=d[a]);for(f&&f(t),o&&o(l);h.length;)h.shift()();return p&&r.push.apply(r,p),n()},l=self.webpackChunk=self.webpackChunk||[];function a(){for(var n,o=0;o<r.length;o++){for(var l=r[o],a=!0,i=1;i<l.length;i++){var u=l[i];0!==e[u]&&(a=!1)}a&&(r.splice(o--,1),n=t(t.s=l[0]))}return 0===r.length&&(t.x(),t.x=e=>{}),n}l.forEach(o.bind(null,0)),l.push=o.bind(null,l.push.bind(l));var i=t.x;t.x=()=>(t.x=i||(e=>{}),(n=a)())})(),t.x()})();
\ No newline at end of file
+/******/ (() => { // webpackBootstrap
```

Comment:
Run in production mode please

Reply:
Sorry, last commit before sleep.  Started implementing maps in the settings.

Reply:
:)